package-lock.json

```diff
@@ -0,0 +1,10955 @@
+{
```

Comment:
@ppcano we use yarn instead of npm in this project. please remove this file.

Reply:
Oops!, sorry, there was so many files with the feature that I didn't realize about this one. 

Removed at https://github.com/ember-cli/ember-cli/pull/8231/commits/a0d59f1a70b7925f570bb07862a2f0ea362e7499